content/docs/traffic-management/session-affinity/consistent-hashing.md

```diff
@@ -219,11 +219,7 @@ To try out session affinity with consistent hashing, you can follow these steps
          group: ""
          kind: Service
      loadBalancer:
-       ringHash:
-         minimumRingSize: 1024
-         maximumRingSize: 2048
-       useHostnameForHashing: true
-       closeConnectionsOnHostSetChange: true
+       maglev: {}
```

Comment:
I realized the problem wasnt with the traffic policy, but that i was using ringhas and it had `useHostnameForHashing: true` which i think was really not jiving with local portforwarding. Maglev worked immediately.